client/shipping-label/view.js

```diff
@@ -187,54 +177,40 @@ class ShippingLabelRootView extends Component {
 		);
 	}
 
-	renderLabelDetails( label, labelNum, index ) {
+	renderLabelDetails( label, labelNum ) {
 		if ( ! label.package_name || ! label.product_names ) {
 			return null;
 		}
 
-		const onMouseEnter = () => this.openTooltip( index );
-		const onMouseLeave = () => this.closeTooltip( index );
-		const onClose = () => this.closeTooltip( index );
-		return (
-			<span>
-				<span className="wcc-metabox-label-item__detail"
-						onMouseEnter={ onMouseEnter }
-						onMouseLeave={ onMouseLeave }
-						ref={ 'label-details-' + index }>
-					{ __( 'Label #%(labelNum)s', { args: { labelNum } } ) }
-				</span>
-				<Tooltip
-					className="wc-connect-popover"
-					isVisible={ this.state.showTooltips[ index ] }
```

Comment:
`<Tooltip>` is a "dumber" component and it needs the `isVisible` prop, but `<InfoTooltip>` (which is in this repo) uses `<Tooltip>` and has internal state. I figured it was cleaner for each individual tooltip to have its own `visible/notVisible` state, instead of the Labels list controlling all the tooltips.